app/src/main/java/com/eventyay/organizer/OrgaApplication.java

```diff
@@ -86,11 +87,20 @@ public void onCreate() {
                     .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(this))
                     .build());
 
-            StrictMode.setThreadPolicy(
-                new StrictMode.ThreadPolicy.Builder()
-                    .detectAll()
-                    .penaltyDeath()
-                    .build());
+            StrictMode.ThreadPolicy.Builder policyBuilder = new StrictMode.ThreadPolicy.Builder()
+                .detectAll()
+                .penaltyDeath();
+
+            if (android.os.Build.VERSION.SDK_INT > Build.VERSION_CODES.O) {
```

Comment:
This should be >=

And why fully qualified class name for Build?

Reply:
You are correct, my mistake, on it.